src/interface/web/app/components/chatInputArea/chatInputArea.tsx

```diff
@@ -270,7 +254,7 @@ export default function ChatInputArea(props: ChatInputProps) {
         }
     }, [recording, mediaRecorder]);
 
-    const chatInputRef = useRef<HTMLTextAreaElement>(null);
+    const chatInputRef = ref as React.MutableRefObject<HTMLTextAreaElement>;
```

Comment:
Getting this error here after this change, consistently on older conversations as well. 

![Screenshot 2024-10-22 at 12 00 39](https://github.com/user-attachments/assets/ca1842a2-4944-4a37-86fc-84b723609f7a)

Reply:
Ouch. Fixed that by adding a null check for chatInputRef as well. Let me know if you're still seeing this issue?